### PR TITLE
interop testing

### PR DESCRIPTION
**Pull request by diasdavid**
seems that `borc` is not doing things with tags the same way that go-cbor does :(

### CODE REVIEW

**Comment by diasdavid** [comment]
@dignifiedquire deserializing a CID TAG from a go serialized object puts an extra zero byte in the front.

**Comment by diasdavid** [comment]
All good now :)

**Comment by diasdavid** [comment]
Only missing is browser testing, but that is a aegir-fixtures issue we are having.

File: test/interop.spec.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+const emptyObjJSON = require('./fixtures/empty-obj.json')
+
+const fooCBOR = loadFixture(__dirname, '/fixtures/foo.cbor')
+const fooJSON = require('./fixtures/foo.json')
+
+const objNoLinkCBOR = loadFixture(__dirname, '/fixtures/obj-no-link.cbor')
+const objNoLinkJSON = require('./fixtures/obj-no-link.json')
+
+const objWithLinkCBOR = loadFixture(__dirname, '/fixtures/obj-with-link.cbor')
+// const objWithLinkJSON = require('./fixtures/obj-with-link.json')
```
Comment (dignifiedquire) [inline test/interop.spec.js]: why u no check me?

**Pull request merged**